src/Matroids/ChowRings.jl

```diff
@@ -0,0 +1,95 @@
+export chow_ring, select
+
+function chow_ring(M::Matroid, extended=false)
+	Flats = flats(M)
+	sizeFlats = size(Flats)[1]
+	n = size(M.groundset)[1]
+	if(!is_loopless(M))
+		throw("Matroid has loops")
+	end
+	if(sizeFlats<2)
+		throw("matroid has to few flats")
+	end
+	proper_flats = Flats[2:sizeFlats-1]
+	var_names = [replace(string("x_",S), "["=>"{", "]"=>"}", ", "=>",") for S in proper_flats]# var names?
```

Comment:
What is this about? I think a comment which says what this line does might be useful... What is `S` here? A vector of `Int` or so?